fda-model-test/ResultTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace fda_model_test
+{
+    public class ResultTest
```

Comment:
good idea. i only had a result test through the integration of it into the simulation class. this is a better strategy!